tools/epanet-output/test/test_epanet_output.cpp

```diff
@@ -0,0 +1,251 @@
+/*
+ *   test_epanet_output.cpp
+ *
+ *   Created: 8/4/2017
+ *   Author: Michael E. Tryby
+ *           US EPA - ORD/NRMRL
+ *
+ *   Unit testing for EPANET Output API using google test.
+*/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string>
+#include "gtest/gtest.h"
```

Comment:
Google Test? were others considered?

Reply:
I am just getting into unit testing. I immediately appreciated the value in running and rerunning tests as part of the development process. I found google test easy to setup using msys2 and easy to use. I am not heavily invested in `gtest` at this point and intend to try other testing frameworks prior to getting "religious" about it.